sbi/utils/user_input_checks_utils.py

```diff
@@ -101,60 +99,6 @@ def variance(self):
         )
 
 
-class ScipyPytorchWrapper(Distribution):
```

Comment:
why remove this?

Reply:
@janfb's assessment was that no one is using this wrapper, so instead of trying to fix this issue he suggested to deprecate it.